keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -58,12 +58,14 @@ class ApplicationController(
       if (includeDetails && !entities.contains("resources")) {
         entities.add("resources")
       }
-      entities.forEach { entity ->
-        results[entity] = when (entity) {
-          "resources" -> applicationService.getResourceSummariesFor(application)
-          "environments" -> applicationService.getEnvironmentSummariesFor(application)
-          "artifacts" -> applicationService.getArtifactSummariesFor(application)
-          else -> throw InvalidRequestException("Unknown entity type: $entity")
+      if (true == results["hasManagedResources"]) {
```

Comment:
You can remove the `true` here :)

Reply:
I can't because the map can return `Any?`.

Reply:
oh I see now, thanks